dockerfiles/Dockerfile.oracle

```diff
@@ -54,9 +71,15 @@ COPY conf/fluent-bit.conf \
     conf/plugins.conf \
     /fluent-bit/etc/
 
+# Copy Fluent-bit binary
+COPY --from=fluentbit-builder /fluent-bit/bin/ /fluent-bit/bin/
+
+# Copy Dependencies
+COPY --from=fluentbit-builder /fluent-bit/rpm/usr/lib /usr/lib
+COPY --from=fluentbit-builder /fluent-bit/rpm/usr/lib64 /usr/lib64
+
```

Comment:
Is there a way to know and copy only required dependency

Reply:
For RPM packages, there are flags --require and `--resolve.` But as we have microdnf in OL9, we are limited to --resolve only. But --resolve, fetches the Transitive Dependency packages too. So we have manually added minimum required packages.  We are justing downloading 4 RPM packages, while with install we get 32-33 packages that has significantly reduced the size.

I guess here you are talking about just copying the required libraries from those downloaded packages. IMO, we can check the runtime dependencies of a binary file using `ldd` . And then we can check if that library exists in /usr/lib, /usr/lib64 and /lib. If not, we can copy it from the downloaded packages. But there are some symlinks too, for that we'll require special handling in our logic. So I just copied the all as of now.

But even If we copy the selected libs, it will not make much difference. Downloaded library(only /usr/lib) size is 
`2.3M`. And out of these mostly gonna get copied.  Max we can reduce 1 M in our image.
As of now Image is like: 
105MB => Base image size
41MB+1.8MB => Binary
3-4MB => library + fluenbit config files.

Reply:
Ok makes sense then.